### ISSUE TITLE
production build issue unresolved './app/app.browser.module.ngfactory''

### ISSUE THREAD

**Issue by artemsoliar** [fulls1z3/universal#39]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => check the README and search github for a similar issue or PR before submitting
[ ] support request => check the README and search github for a similar issue or PR before submitting
[ ] feature request
```

**Current behavior**
It seems to be that recently an npm dependency has been updated which causes build issues for production build  'build:universal-prod'
```
Error: ./src/client/main-browser.ts
Module not found: Error: Can't resolve './app/app.browser.module.ngfactory' in 'C:\universal-master\src\client' 
```

**Expected/desired behavior**
Prod Build should compile the production bundles

**Minimal reproduction of the problem with instructions**
Clone this repository locally
Run
```
npm install
npm run build:universal-prod
```

Please tell us about your environment:
Windows 10

**Comment by artemsoliar** [comment]
after npm i enhanced-resolve@3.3.0 I can build in production 
same problem https://github.com/angular/angular-cli/issues/7113

**Comment by theomathieubhvr** [comment]
Thanks @artemSoliar !

**Comment by theomathieubhvr** [comment]
Fixed by 8eb90cfca5d98da2ee2cd349ac6048238604c551

**Comment by artemsoliar** [comment]
This is a temporary solution only when it's installed as an addition into an already up and running project.  Deployment through a fresh install with npm install with this dependency set in package.json does not work.
https://github.com/angular/angular-cli/issues/7113#issuecomment-317437131

**Comment by theomathieubhvr** [comment]
You're right that's why my PR does not work 
https://github.com/ng-seed/universal/pull/41

**Comment by theomathieubhvr** [comment]
Now fixed via #41 (downgraded to webpack 3.3.0)

**Comment by fulls1z3** [comment]
@artemSoliar I'll publish a fix asap.
